### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by Iron-Ham**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -17,3 +17,5 @@ beans beans beans bean
 goya? no thankya
 
 beaaaaan
+
+who is the best bean boi
```
Comment (Iron-Ham) [inline readme.md:21]:
```suggestion
who is the best bean boy
```

**Comment by Iron-Ham** [comment]
Hmmmm

File: readme.md [hunk 2]
```diff
@@ -17,3 +17,5 @@ beans beans beans bean
 goya? no thankya
 
 beaaaaan
+
+who is the best bean boy
```
Comment (Iron-Ham) [inline readme.md:21]: 🤔

**Review by Iron-Ham — COMMENTED** [review]
Very

File: readme.md [hunk 3]
```diff
@@ -17,3 +17,5 @@ beans beans beans bean
 goya? no thankya
 
 beaaaaan
+
```
Comment (Iron-Ham) [inline readme.md:20]: is this

File: readme.md [hunk 4]
```diff
@@ -18,4 +18,4 @@ goya? no thankya
 
 beaaaaan
 
-who is the best bean boi
+who is the best bean boy
```
Comment (Iron-Ham) [inline readme.md:21]: i

File: readme.md (hunk 3 above)
Comment (selarun13) [reply readme.md:20]: 0x6F99E90Dd385c23826b416528e325214671FeBf6 my adres

**Review by selarun13 — APPROVED** [review]
0x6F99E90Dd385c23826b416528e325214671FeBf6

**Review by The-Tru-th — COMMENTED** [review]
This is a bug

**Pull request closed without merging**